net/pdns-recursor/Makefile

```diff
@@ -0,0 +1,63 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pdns-recursor
+PKG_VERSION:=4.1.14
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://downloads.powerdns.com/releases/
+PKG_HASH:=7fceb8fa3bea693aad49d137c801bb3ecc15525cc5a7dc84380321546e87bf14
+
+PKG_MAINTAINER:=James Taylor <james@jtaylor.id.au>
+PKG_LICENCE:=GPL-2.0-only
+PKG_LICENCE_FILES:=COPYING
+PKG_CPE_ID:=cpe:/a:powerdns:recursor
+PKG_INSTALL:=1
+
+PKG_FIXUP:=autoreconf
+
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/pdns-recursor
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=IP Addresses and Names
+  USERID:=pdns:pdns
+  TITLE:=PowerDNS Recursor
+  DEPENDS:=+libatomic +boost +boost-context +boost-system +boost-thread +libatomic +liblua +libopenssl +protobuf
```

Comment:
boost-thread and boost-system can be removed as they are implied by boost-context.

Reply:
Removed these and the duplicate libatomic